pkg/controllers/provisioning/scheduling/volumetopology.go

```diff
@@ -208,11 +179,10 @@ func (v *VolumeTopology) validateVolume(ctx context.Context, volumeName string)
 	return nil
 }
 
-func (v *VolumeTopology) validateStorageClass(ctx context.Context, storageClassName *string) error {
-	// we have a storage class name, so ensure that it exists
-	if ptr.StringValue(storageClassName) != "" {
+func (v *VolumeTopology) validateStorageClass(ctx context.Context, storageClassName string) error {
+	if storageClassName != "" {
```

Comment:
When will this ever be an empty string now that we are checking that the storage class name can't be empty?

Reply:
Good point, this check can be removed 👍